metacat-main/src/main/resources/application.yml

```diff
@@ -46,3 +47,5 @@ spring:
     default-property-inclusion: always
     deserialization:
       FAIL_ON_UNKNOWN_PROPERTIES: false
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
If you can avoid needing to set this, that'd be for the best long term. But not sure what specifically caused you to need this.

Reply:
Yeah, it looks like we unintentionally override beans in a few places. I added this as a short-term fix for Boot 2.1 since it's false by default there. Shouldn't be necessary for Boot 2.0. I'll make sure to fix the overrides when we upgrade to 2.1.